config/certmanager/certificate.yaml

```diff
@@ -0,0 +1,25 @@
+# The following manifests contain a self-signed issuer CR and a certificate CR.
```

Comment:
are you using certmanager? If not, probably remove this directrory else it's probably better to use https://docs.openshift.com/container-platform/4.6/security/certificates/service-serving-certificate.html

Reply:
certmanager isn't used. I'll remove this.